### PR TITLE
Issue20 shacl

### PR DESCRIPTION
**Pull request by CunliangGeng**
To solve issue #20 .

### CODE REVIEW

File: fdp/validator.py [hunk 1]
```diff
@@ -13,9 +21,13 @@ def _validate(data, shapes_file, fdp=False):
         s_set = set([s for s, p, o in g])
         if len(s_set) == 0:
             raise ValueError('Empty content in metadtata')
```
Comment (c-martinez) [inline fdp/validator.py]: metadtata -> metadata

File: fdp/validator.py [hunk 2]
```diff
@@ -13,9 +21,13 @@ def _validate(data, shapes_file, fdp=False):
         s_set = set([s for s, p, o in g])
```
Comment (c-martinez) [inline fdp/validator.py]: `set([s for s, p, o in g])` could also be written `set(g.subjects()))` -- and is perhaps more expressive?

**Review by c-martinez — APPROVED** [review]
Two small details, but you can ignore them if you don't think they are relevant.

**Comment by CunliangGeng** [comment]
Thanks, nice catch! Updated and merge now.

**Pull request merged**